templates/spin_sidebar.hbs

```diff
@@ -13,6 +13,8 @@
                             href="{{site.info.base_url}}/spin/install">Install</a></li>
                     <li><a {{#if (active_content request.spin-path-info "/spin/quickstart" )}} class="active" {{/if}}
                             href="{{site.info.base_url}}/spin/quickstart">Quickstart</a></li>
+                    <li><a {{#if (active_content request.spin-path-info "/spin/designing-spin-apps.md" )}} class="active" {{/if}}
```

Comment:
Don't put this in the Get Started section under its current title.  If we want to split the "showcase" / "see what other people are building" content from the "how to design larger applications" content (which is what the title reflects but does not currently exist) then we could put the former in Get Started.  The as-yet-unwritten app design stuff will belong under Applications though; it's definitely not starter material!

(I realise you've changed the title here but it needs to change on the page too, and I'd suggest changing the filename because we still want a "designing apps" page.)

Reply:
Closing this in favour of splitting the idea into two separate files. One to see what people are building with Spin and the other for explaining how to design and create large applications with Spin.